pages/posts.js

```diff
@@ -0,0 +1,42 @@
+import getPosts from '../src/Services/get-posts';
+import Form from '../src/Components/Form/Form';
+import AddFormButton from '../src/Components/Form/AddFormButton';
+import { saveToLocal, loadFromLocal } from '../src/Components/lib/localStorage';
+import { useState, useEffect } from 'react';
+import dynamic from 'next/dynamic';
+
+export function getStaticProps() {
+	const data = getPosts();
+
+	return {
+		props: {
+			data,
+		},
+	};
+}
+export default function Posts({ data }) {
+	const [addPost, setAddPost] = useState(loadFromLocal('localPosts') ?? data);
```

Comment:
Why is the state called `addPost`? I'd actually expect a function behind this name which makes adding a post possible. `posts` might be a more suitable name?

Reply:
LGTM 🚀